Release/src/http/client/http_client_asio.cpp

```diff
@@ -579,8 +580,6 @@ class asio_context final : public request_context, public std::enable_shared_fro
 
             request_stream << CRLF;
 
-            m_context->m_timer.start();
-
```

Comment:
This call moved to the call site of the surrounding method (`ssl_proxy_tunnel::start_proxy_connect()`) i.e. `asio_context::start_request()`. This is the only location where `ssl_proxy_tunnel::start_proxy_connect()` is called if a proxy-tunnel is required. `asio_context::start_request()` also initiates the ordinary connection (without a proxy-tunnel). Therefore, I found it more consistent to move the call to `start_connecting()` there.